example_test.py

```diff
@@ -72,12 +73,6 @@ def test_create_file(self):
 
     def test_delete_file(self):
         """Test example.delete_file()
-
-        `self.fs.CreateFile()` is convenient because it automatically creates
-        directories in the fake file system and allows you to specify the file
-        contents.
-
-        You could also use `open()` or `file()`.
```

Comment:
Indeed we should eliminate references to `file()` in the documentation because it is gone in Python 3.